source/heatfluxClass.py

```diff
@@ -984,7 +1013,10 @@ def getHFprofile(self, PFC, mafotPsi=False):
             q *= q0
             print("Tophat lqCN: {}".format(self.lqCN))
             log.info("Tophat lqCN: {}".format(self.lqCN))
-
+        
+        #rzq profile
+        if self.hfMode == 'rzqprofile':
```

Comment:
```suggestion
        elif self.hfMode == 'rzqprofile':
```

Reply:
I think you should add a print statement here or somewhere else that tells the user that because they are using the rzqprofile the result will not conserve energy.  because there is no q0 term, the PSOL will not equal P_target.  Adding a comment here for future users, and also printing a statement ensures people see this.

Reply:
Added more explanation at the comment